CMakeLists.txt

```diff
@@ -65,6 +66,7 @@ file(GLOB UtilTestsGlob unit_tests/util/*.cpp)
 add_library(UTIL OBJECT ${UtilGlob})
 add_library(EXTRACTOR OBJECT ${ExtractorGlob})
 add_library(CONTRACTOR OBJECT ${ContractorGlob})
+add_library(STORAGE OBJECT ${StorageGlob})
 add_library(ENGINE OBJECT ${EngineGlob})
 add_library(SERVER OBJECT ${ServerGlob})
 add_library(GRAPH OBJECT src/extractor/external_memory_node.cpp)
```

Comment:
Hmm I'm only now seeing this, but do we really need the `GRAPH` library, for a handful of lines of code in `external_memory_node.cpp`? Would love to just put it into a header, remove the library and call it a day.